### ISSUE TITLE
viewBox in SVG but generate viewbox insteadof

### ISSUE THREAD

**Issue by ZhenTek** [website-scraper/node-website-scraper#369]
### Configuration
**version:** [`-- website-scraper@4.0.1]

```
<!-- Note that if you don't provide urls which we can use to reproduce problem - in most of cases we will not be able to help and issue will be closed. -->

### Description

[Description of the issue]

**Expected behavior:** [Attribute name should be viewBox in SVG tag]

**Actual behavior:** [Attribute name is viewbox, lowercase]

### Additional Information

[Any additional information, configuration or data that might be necessary to reproduce the issue]

**Comment by ZhenTek** [comment]
it works fine if enable xmlMode in cheerio lib, is this can be customized?

**Comment by ZhenTek** [comment]
Alternative method is update the cheerio to the latest version(1.0.0-rc.3, not stable), but HTML entities cannot generate correctly, like &trade;  or  &copy;

**Comment by s0ph1e** [comment]
Hi @ZhenTek 

Thank you for reporting issue and suggestion how to fix it 👍
I'll check this cheerio options

Could you please share html element example which causes issue?

**Comment by ZhenTek** [comment]
The attribute viewBox in svg tag, after retrieve it becomes viewbox with lower case

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by s0ph1e** [comment]
Fix release in version 4.1.0
